src/Lecture1.hs

```diff
@@ -81,7 +83,11 @@ and 1 is the smallest, and 7 - 1 = 6.
 Try to use local variables (either let-in or where) to implement this
 function.
 -}
-minmax x y z = error "TODO"
+minmax :: Int -> Int -> Int -> Int
+minmax x y z = 
+    let max = maximum [x, y, z]
```

Comment:
I see that GHC outputs some warnings ⚠️
In this case, it happens because `max` and `min` are the names of standard functions in Prelude. It still works due to name shadowing. But generally, it's better to avoid name conflicts with standard functions.

It's a bit tough to do this here because both `max` and `maximum` are already taken 😅 But I would use smth like `mx` or `maxVal` (and similar for minimum).